x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -290,16 +290,17 @@ export class BlendedVectorLayer extends VectorLayer implements IVectorLayer {
   async syncData(syncContext: DataRequestContext) {
     const dataRequestId = ACTIVE_COUNT_DATA_ID;
     const requestToken = Symbol(`layer-active-count:${this.getId()}`);
-    const searchFilters: VectorSourceRequestMeta = await this._getSearchFilters(
+    const searchFilters: VectorSourceRequestMeta = await this._getVectorSourceRequestMeta(
```

Comment:
Since you are renaming the method, how about renaming the variable as well, from searchFilters to requestMeta?